lib/meow/report.ex

```diff
@@ -105,6 +105,17 @@ defmodule Meow.Report do
 
   alias VegaLite, as: Vl
 
+  @spec metrics_plots(t()) :: list(VegaLite.t())
```

Comment:
```suggestion
```

Nit: `metrics_plots` -> `metric_plots`
Uber nit: I'd move it under the `plot_metrics` where it's used for the first time :)